knowledge_repo/app/utils/emails.py

```diff
@@ -88,16 +89,27 @@ def send_subscription_email(post, tag):
         return
 
     default_recipients = ['knowledge_consumer@notreal.com']
-    subject = u"New KR post: {}".format(post.title)
+    subject = u"New knowledge post with tag [{}]!".format(tag.name)
```

Comment:
To clarify why I removed the tag from the subject, I will describe a use case (please remember that a post can have more than one tag assigned):

1. A user is subscribed to tags `tag1` and `tag2`.
1. A post with the tags `tag1` and `tag2` is published.
1. The user described in point `1` receives and email with the subject `New knowledge post with tag [tag1]!`.

I found the following weird things about it:

1. Only one of the two tags was included in the subject and the full list of tags was not included in the email either. 
1. The subject of the email is not relevant at all.

That's why I...:

1. Included the full list of the tags in the new HTML body .
1. Used the post title as email subject for  better readability and identification purposes.
1. Included a prefix in the subject so the user can easily recon that the email was received from the Knowledge Repo. I used `New KR post:` to achieve that but I agree with @serenajiang that acronym could be ambiguous, so I think it is better to use `Knowledge post:`, `New knowledge post:` or similar.

Reply:
Good points. I'll change it to `New knowledge post: ...`. Thanks!